paper-radio-button.html

```diff
@@ -87,38 +89,30 @@
          * @event iron-change
          */
 
-        /**
-         * Gets or sets the state, `true` is checked and `false` is unchecked.
-         */
-        checked: {
-          type: Boolean,
-          value: false,
-          reflectToAttribute: true,
-          notify: true,
-          observer: '_checkedChanged'
-        },
-
-        /**
-         * If true, the button toggles the active state with each tap or press
-         * of the spacebar.
-         */
-        toggles: {
-          type: Boolean,
-          value: true,
-          reflectToAttribute: true
-        }
+         ariaActiveAttribute: {
+           value: 'aria-checked'
+         }
       },
 
       attached: function() {
-        var trimmedText = Polymer.dom(this).textContent.trim();
-        if (trimmedText === '') {
-          this.$.radioLabel.hidden = true;
-        }
+        this._isReady = true;
```

Comment:
Yes 😢

`_buttonStateChanged` (see L122) is still called before `attached`, and stomps over the user-set `checked` attribute. :sob: